src/test/java/org/candlepin/subscriptions/clowder/ClowderJsonEnvironmentPostProcessorTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.clowder;
+
+import static org.candlepin.subscriptions.clowder.ClowderJsonEnvironmentPostProcessor.JSON_RESOURCE_LOCATION;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Collections;
+import java.util.function.Supplier;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.logging.DeferredLogFactory;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.MapPropertySource;
+import org.springframework.core.env.PropertySource;
+import org.springframework.core.env.StandardEnvironment;
+import org.springframework.test.context.support.TestPropertySourceUtils;
+import org.springframework.web.context.support.StandardServletEnvironment;
+
+class ClowderJsonEnvironmentPostProcessorTest {
```

Comment:
I can do that, but currently it would be the only file there.  `swatch-core` doesn't even have a `src/test` directory right now.  I looked into moving all the relevant tests into that directory at one point but it's not just a `mv` command; there are a lot of interdependencies that will need to be sort out.

Reply:
I'd rather start the process now, if for this single test it's a simple `mv`. Otherwise, let's just handle the whole bulk in one go w/ a tech-debt card.